src/modules/contest/standings/standings.ts

```diff
@@ -0,0 +1,18 @@
+import { CodeforcesClient } from '../../../CodeforcesClient';
+import { ICodeforcesClient } from '../../../interfaces/codeforces-client.interface';
+import { Standings } from '../../../interfaces/standings.interface';
+
+type ContestStandingsParams = {
+  contestId: string;
+  from?: string;
+  count?: number;
+  handles?: string;
```

Comment:
Please don't forget to add a JSDoc comment here to provide the guidance in the code editor.

```
/**
 * Semicolon-separated list of handles. No more than 10000 handles are accepted.
*/
```

JSDoc comments are special comments that allows you to add descriptions and info about the code. The IDEs parse those comments to provide auto completion and guidance on hovering on the code.

If you want to have an in-depth look you can see this website: https://jsdoc.app/about-getting-started.html